dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -140,6 +142,10 @@ public void validate(Object target, Errors errors) {
         if (masterConfig.getMaxCpuLoadAvg() <= 0) {
             masterConfig.setMaxCpuLoadAvg(Runtime.getRuntime().availableProcessors() * 2);
         }
+        if (masterConfig.getWorkerGroupRefreshInterval().getSeconds() <= 0) {
```

Comment:
Should we set a threshold to avoid that users set a too small interval and cause heavy load to database? For example if the users set to 1s and master loads the workers every second, which can be too frequent. 

This is for the sake of performance.

Reply:
Agree with you, I set this value cannot smaller than 10s.